clang/lib/DPCT/ExtReplacements.cpp

```diff
@@ -319,14 +319,16 @@ void ExtReplacements::addReplacement(std::shared_ptr<ExtReplacement> Repl) {
   if (isInvalid(Repl, FileInfo))
     return;
   if (Repl->getLength()) {
-    if (Repl->IsSYCLHeaderNeeded())
+    if (Repl->isSYCLHeaderNeeded())
```

Comment:
Suggest not mix refactor, NFC update with logic fix or new feature patch together.  For code refactor,  please summmit specific PR for it.